### PR TITLE
Adds Kubernetes Göteborg as meetup

### PR DESCRIPTION
**Pull request by Solisol**

### CODE REVIEW

File: gothenburg/meetup.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+- address: Östra Hamngatan 50B
+  attendees: 36
+  date: "2017-10-25T17:00:00Z"
+  duration: 2h0m0s
+  id: 242830009
+  name: Metrics och alarm med Prometheus
+  presentations:
+  - duration: 60m0s
+    slides: ""
+    speakers: null
```
Comment (luxas) [inline gothenburg/meetup.yaml]: please add Karl to speakers.yaml and reference him here

File: gothenburg/meetup.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  attendees: 14
+  date: "2017-08-30T17:00:00Z"
+  duration: 1h0m0s
+  id: 241821276
+  name: Var med och sänk ett kluster!
+  presentations:
+  - duration: 60m0s
+    slides: ""
+    speakers: null
+    title: Staffan Olsson - Vad händer när 100 tjänster delar på 10 noder?
```
Comment (luxas) [inline gothenburg/meetup.yaml]: please add Staffan to speakers.yaml and reference him here

File: gothenburg/meetup.yaml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  sponsors:
+    other: null
+    venue: null
+name: Kubernetes Göteborg
+organizers:
+- jessicaandersson
```
Comment (luxas) [inline gothenburg/meetup.yaml]: please add the other organizers here, too

File: gothenburg/meetup.yaml [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    - federicohernandez
+    title: Federico Hernandez - Knowing what your Kubernetes cluster is doing
+  - delay: 5m0s
+    duration: 10m0s
+    slides: ""
+    speakers:
+    - simonesciarrati
+    title: Simone Sciarrati - Zero downtime upgrades of Kubernetes
+  sponsors:
+    other: null
```
Comment (luxas) [inline gothenburg/meetup.yaml]: add CNCF?

File: gothenburg/meetup.yaml [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    slides: ""
+    speakers:
+    - jessicaandersson
+    title: Getting Developers to Adopt Your Service
+  - delay: 5m0s
+    duration: 30m0s
+    slides: ""
+    speakers:
+    - federicohernandez
+    title: Federico Hernandez - Knowing what your Kubernetes cluster is doing
```
Comment (luxas) [inline gothenburg/meetup.yaml]: please drop the speaker's name

File: gothenburg/meetup.yaml [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+  - delay: 5m0s
+    duration: 10m0s
+    slides: ""
+    speakers:
+    - simonesciarrati
+    title: Simone Sciarrati - Zero downtime upgrades of Kubernetes
```
Comment (luxas) [inline gothenburg/meetup.yaml]: please drop the speaker's name

File: gothenburg/README.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+## Meetups organized in Gothenburg
```
Comment (luxas) [inline gothenburg/README.md]: this file is autogenerated, generate it with `make generate`

File: gothenburg/meetup.yaml [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+  name: 18 months of kubernetes in production and learning
+  presentations:
+  - duration: 30m0s
+    slides: ""
+    speakers:
+    - jessicaandersson
+    title: Getting Developers to Adopt Your Service
+  - delay: 5m0s
+    duration: 30m0s
+    slides: ""
```
Comment (luxas) [inline gothenburg/meetup.yaml]: can you please populate slides for as many of these talks as you can?

File: gothenburg/meetup.yaml [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+- address: Östra Hamngatan 50B
+  attendees: 33
+  date: "2018-11-21T17:00:00Z"
+  duration: 2h0m0s
+  id: 256374545
+  name: Serverless med Knative
+  presentations:
+  - duration: 60m0s
+    slides: ""
+    speakers: null
```
Comment (luxas) [inline gothenburg/meetup.yaml]: please add Staffan to speakers.yaml and reference him here

File: gothenburg/meetup.yaml [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+    title: Staffan Olsson - Demo av Knative
+  sponsors:
+    other: null
+    venue: null
```
Comment (luxas) [inline gothenburg/meetup.yaml]: no one sponsored this meetup?

File: gothenburg/meetup.yaml (hunk 4 above)
Comment (Solisol) [reply gothenburg/meetup.yaml]: well, not really yet :) For next meetup, sure!

File: gothenburg/meetup.yaml (hunk 10 above)
Comment (Solisol) [reply gothenburg/meetup.yaml]: Not that I can easily find, if possible I hope to be able to add information about it later on.

**Pull request merged**